activemodel/CHANGELOG.md

```diff
@@ -13,5 +13,11 @@
     `config.active_support.halt_callback_chains_on_return_false` option, will
     either not work at all or display a deprecation warning.
 
+*   Change validates_acceptance_of to accept true by default.
+
+    The default for validates_acceptance_of is now "1" and true.
+    In the past, only "1" was the default and you were required to add 
+    accept: true.
+
```

Comment:
@mokhan Please add CHANGELOG at the top of the file. New entries get at the top.